Libraries/LibJS/Tests/Array.prototype.slice.js

```diff
@@ -0,0 +1,53 @@
+load("test-common.js");
+
+try {
+    assert(Array.prototype.concat.length === 1);
```

Comment:
```suggestion
    assert(Array.prototype.slice.length === 1);
```

Reply:
thanks, it was because of ctrl+c and ctrl+v. :)